protoconvert.go

```diff
@@ -69,9 +70,12 @@ func valueToProto(value any) (*pb.Value, error) {
 		return &pb.Value{Inner: &pb.Value_Int{Int: v.UnixMilli()}}, nil
 	case time.Time:
 		return &pb.Value{Inner: &pb.Value_Int{Int: v.UnixMilli()}}, nil
+	case *big.Int:
+		return &pb.Value{Inner: &pb.Value_Varint{Varint: &pb.Varint{
+			Value: v.Bytes(),
```

Comment:
It looks like this marshal code doesn't handle negative values correctly. Tried `big.NewInt(-64)`:

```
cassandra@cqlsh> SELECT varint_col FROM test.all_types ;

 varint_col
------------
         64

(1 rows)
```

Here's more information on the wire format: https://github.com/apache/cassandra/blob/cassandra-4.0.5/doc/native_protocol_v3.spec#L861-L879. The wire format uses a two's complement representation and the `Int.Bytes()` method only returns the absolute value of the integer without a sign (it's in another field). gocql is a good reference for handling this: https://github.com/gocql/gocql/blob/5cf3afd187a9244635fc2ca7a7419465a4bffd06/marshal.go#L1196

Reply:
Fixed.

Reply:
How's this?